DragonSlayer.h

```diff
@@ -1,8 +1,17 @@
 #pragma once
 
+#include "Character.h"
+#include "AttackItem.h"
+
 struct DragonSlayer : Character
 {
+    DragonSlayer(std::string name_, int hp_, int armor_, int attackDamage_ = 4);
+    
+    const std::string& getName() override;
+    std::string getStats() override;
     void attack(Character& other) override;
+
 private:
     const std::string name;
-};
\ No newline at end of file
+    std::unique_ptr<Item> attackItem { new AttackItem() }; // why private!???
```

Comment:
It's private because no other classes need to know what this member variable exists.